src/lib/fuzzySearch.js

```diff
@@ -0,0 +1,17 @@
+const identity = thing => thing
+
+const fuzzySearch = (array, string, lookedUpProperty = false) => {
+  const words = string.split(' ')
+  const regexes = words.map(word => new RegExp(`(${word})`, 'ig'))
+  const result = array.reduce(
+    (acc, item) => {
+      const itemValue = lookedUpProperty ? item[lookedUpProperty] : item
+      const regexRes = regexes.map(regex => itemValue.match(regex))
```

Comment:
return regexes.some(regex => !itemValue.match(regex)) ? acc : acc.concat(item)
shorter + will stop looping as soon as a non-match is found

(I think, written super quickly before I lost connection in the train)